Makefile

```diff
@@ -125,7 +125,7 @@ endif
 ## Editor navigation files
 C_SOURCE_FILES = $(wildcard include/*/*.h library/*.[hc] programs/*/*.[hc] tests/suites/*.function)
 tags: $(C_SOURCE_FILES)
-	ctags -o $@ $(C_SOURCE_FILES)
+	ctags -o $@ --langmap=c:+.h.function $(C_SOURCE_FILES)
```

Comment:
Damn, the Emacs ctags and exuberant-ctags are not compatible. And the [BSD](https://www.freebsd.org/cgi/man.cgi?query=ctags&apropos=0&sektion=0&manpath=FreeBSD+12.1-RELEASE+and+Ports&arch=default&format=html) ctags is again different.

Can you please change to
```
# Exuberant-ctags invocation. Other ctags implementations may require different options.
CTAGS = ctags --langmap=c:+.h.function -o
tags: $(C_SOURCE_FILES)
	$(CTAGS) $@ $(C_SOURCE_FILES)
```
so people can run `make CTAGS='ctags -f' or whatever works for them?

Reply:
Thanks! I used your suggestion.